docs/Examples.md

```diff
@@ -16,6 +24,14 @@ To tear down the above just:
 docker-compose down
 ```
 
+Next up, we will start several services that connect to each other as a chain using different communication
+stacks. All [patron services](Architecture.md#Service) start an HTTP exposing 'liveness', 'readyness', 'metrics' and 'debugging'
+endpoints, therefore, all the following 'microservices' use different ports in order to avoid collisions.
+Below you can find a simplified view of the communication between the services:
+
+[comment]: <> (image is the result of the plantuml diagram with the same name)
+![Flow of requests/events](images/example-arch.png)
```

Comment:
This simple diagram is meant to give a visual input on how the whole integration suite communicates.